packages/phone-number-privacy/common/package.json

```diff
@@ -16,10 +16,17 @@
     "lib/**/*"
   ],
   "dependencies": {
-    "@celo/utils": "^0.1.20-dev",
-    "bignumber.js": "^9.0.0"
+    "@celo/contractkit": "0.4.14-dev",
+    "@celo/utils": "0.1.20-dev",
+    "bignumber.js": "^9.0.0",
+    "elliptic": "6.5.3",
+    "@types/elliptic": "^6.4.12",
+    "firebase-functions": "^3.6.0"
```

Comment:
Can we avoid a dep on firebase-functions? This will add this dep to the signer which doesn't need it.

Reply:
good point.. we're just using `request` which is from express, so we can just use express as a dependency.